spring-2013/hw1/src/CreditCardValidator.hs

```diff
@@ -0,0 +1,22 @@
+module CreditCardValidator
+    ( toDigits
+    , doubleEveryOther
+    , sumDigits
+    , validate
+    ) where
+
+toDigits :: Integer -> [Integer]
+toDigits n
+    | n <= 0 = []
+    | otherwise = map readChar . show $ n
```

Comment:
yeah, good call. I think this is a holdover from the fact that this _used_ to be a pointfree function before I needed to check the value of `n` which screwed everything up.